### ISSUE TITLE
Naming of model 

### ISSUE THREAD

**Issue by hlevring** [yuyun2000/SpeechDenoiser#9]
As I understand your model denoiser_model.onnx is a repackaged version of
DeepFilterNet3_onnx.tar.gz ,  so  to make that more clear then maybe it can make sense to give it a name starting with  
DeepFilterNet3 xxx. onnx  or something?

**Comment by yuyun2000** [comment]
Yes, of course, I just think the name is a bit long. If you want, you can submit a PR to modify the name or content of all files.
